test/e2e/specs/data-json.js

```diff
@@ -0,0 +1,289 @@
+import { expect } from 'chai'
+import sandbox from '../helpers/sandbox'
+
+const Backendless = sandbox.Backendless
+
+describe('Data - JSON', function() {
+  let TestTable
+
+  let savedObj
+
+  let jsonUpdateBuilder
+
+  sandbox.forSuite()
+
+  const TABLE_NAME = 'TestTableWithJSON'
+
+  before(async function() {
+    TestTable = Backendless.Data.of(TABLE_NAME)
+
+    await this.tablesAPI.createTable(TABLE_NAME)
+    await this.tablesAPI.createColumn(TABLE_NAME, 'myJson', this.tablesAPI.DataTypes.JSON)
+  })
+
+  beforeEach(async function() {
+    jsonUpdateBuilder = Backendless.JSONUpdateBuilder
+
+    savedObj = await TestTable.save({
+      name: 'test', age: 30, myJson: {
+        'letter'     : 'a',
+        'number'     : 10,
+        'decimals'   : [12.3, 43.28, 56.89],
+        'colours'    : ['red', 'green', 'blue'],
+        'description': 'It is an "Example".',
+        'timeMarks'  : {
+          'time'     : '12:18:29.000000',
+          'date'     : '2015-07-29',
+          'date_time': '2015-07-29 12:18:29.000000'
+        }
+      }
+    })
+  })
+
+  it('set', async () => {
+    const jsonUpdateSet = jsonUpdateBuilder.SET()
+      .addArgument('$.letter', 'b')
+      .addArgument('$.number', 36)
+      .addArgument('$.state', true)
+      .addArgument('$.colours[0]', null)
+      .addArgument('$.innerObject', { a: 'b' })
+      .addArgument('$.innerArray', [4, 3, 2])
+      .toJSON()
```

Comment:
remove the `toJSON` method, it will be called automatically in the BackendlessRequest

Reply:
+